discovery-infra/tests/test_auth.py

```diff
@@ -0,0 +1,282 @@
+import pytest
+import random
+from contextlib import suppress
+from string import ascii_lowercase
+from typing import Optional
+from tests.base_test import BaseTest
+from tests.conftest import env_variables
+from assisted_service_client.rest import ApiException
+from test_infra import assisted_service_api, consts, utils
+
+def random_name():
+    return ''.join(random.choice(ascii_lowercase) for i in range(10))
+
+
+@pytest.fixture()
+def api_client():
```

Comment:
Subsequent PR will unify this with fixture from conftest and update existing tests